src/ui/settings/index.js

```diff
@@ -29,7 +29,8 @@ export default function setupSettings(kbnServer, server, config) {
     return client
       .get({ ...clientSettings })
       .then(res => res._source)
-      .then(user => hydrateUserSettings(user));
+      .then(user => hydrateUserSettings(user))
+      .catch(e => { return {}; });
```

Comment:
Good call on moving the `.catch` call one level up. Since we are also going to be logging here, I'm just going to create a small function to do that and return {}, and pass that function to `.catch`. Hopefully this will help make it clear _why_ this bit of code exists.